docs/dev-concepts/participant-authentication.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_label: Participant authentication
+sidebar_position: 5
+---
+
+# Participant authentication with XMTP
+
+<!--What do we think about the title of this topic? Consider this definition of "participant" - "Topics have participants, but not users. Client apps and the XMTP network have users, but not participants." Based on this definition I don't think we are authenticating participants, are we? I am thinking of "User authentication" (friendly language) or "Blockchain account authentication" (less friendly, but more accurate?)-->
+
+<!--I removed the language about relationships between XMTP identities being established using a set of keys because the XMTP identity IS one of these keys. The way this is worded, if the relationship is established by the keys, the relationship must be between something that exists before key generation. Following the comment above, I think the keys enable establishing relationships between users (sender and recipient), or more specifically, the sender blockchain account and recipient blockchain account. Any thoughts?-->
+
+<!--In the first sentence below, is it the XMTP client SDK that enables this - or the client/client protocol? I think the client provides this functionality based on the client protocol and the client SDK exposes it? What might make most sense to a developer reader here? I ask because I am seeing the opportunity, or need, the make a clearer division in the docs between the protocol and client -- and the SDK. For example, many of these topics in the Development concepts section are specifically about the protocols and could be refined to make this clearer.-->
```

Comment:
I agree, I'd say it's the protocol, the SDK is an implementation of this protocol.